pkg/plan/resource/kubeadm_join.go

```diff
@@ -35,6 +36,8 @@ type KubeadmJoin struct {
 	IgnorePreflightErrors []string `structs:"ignorePreflightErrors"`
 	// External Load Balancer name or IP address to be used instead of the master's IP
 	ExternalLoadBalancer string `structs:"externalLoadBalancer"`
+	// Kubernetes Version is used to prepare different parameters
```

Comment:
Again this appears to be used as the kubeadm version

Reply:
It's Kubernetes version passing thru `CreateSeedNodeSetupPlan`, and it's not the version of Kubeadm. I'll make it clear.